pyccl/halos/profiles.py

```diff
@@ -919,9 +1112,18 @@ def __init__(self, mass_bias=0.8, P0=6.41,
         self._fourier_interp = None
         super(HaloProfilePressureGNFW, self).__init__()
 
-    def update_parameters(self, mass_bias=None, P0=None,
-                          c500=None, alpha=None, beta=None, gamma=None,
-                          alpha_P=None, P0_hexp=None, x_out=None):
+    def update_parameters(
```

Comment:
For example, I find this choice of indenting harder to read than what was here before. Like I said, both are fine by flake8, but I'd leave it as it was for consistency. Python packages like Numpy or Matplotlib don't use this kind of indentation (which is more C-like).